tests/test_template.py

```
@@ -494,3 +496,97 @@ def test_generated_github_workflows_match_act_validation_contract(
+    [
+        ("mutation-312-pure", False, "3.12", False),
+        ("mutation-312-rust", True, "3.12", False),
+        ("mutation-313-pure", False, "3.13", True),
+        ("mutation-314-rust", True, "3.14", True),
+    ],
+)
+def test_generated_mutation_testing_gating(
+    copier: CopierFixture,
+    tmp_path: Path,
+    target_dir: str,
+    use_rust: bool,
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)